tools/Makefile

```diff
@@ -1,5 +1,5 @@
-.PHONY: clean
-clean:
+.PHONY: e2eclean
```

Comment:
question: Why is this changing?

Reply:
whoa, I remember seeing the rule called "e2eclean" but the .PHONY didn't match it, so I changed the .PHONY.  But the first change must have been me typing into the wrong terminal or something.